sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,16 +1,44 @@
 # Release History
 
+## 3.0.0 (2022-09-19)
+
+### Features Added
+
+- Added `additionallyAllowedTenants` to the following credential options to force explicit opt-in behavior for multi-tenant authentication via the `MultiTenantTokenCredentialOptions` to the following:
```

Comment:
maybe mention the credentials impacted rather than their named option bags? I feel like nobody uses these types directly